Jenkinsfile_CNP

```diff
@@ -287,37 +289,55 @@ def packerBuildProd(body) {
 
 }
 
-def initialiseLogstashConfFiles(KeyVault keyVault) {
-
-        db_host = keyVault.find("ccd-data-store-api-POSTGRES-HOST").trim()
+def initialiseLogstashConfFiles(KeyVault keyVault, String env) {
+        db_host = keyVault.find("${dataBasetName}-POSTGRES-HOST").trim()
         echo "retrieved db host: ${db_host}"
-        db_port = keyVault.find("ccd-data-store-api-POSTGRES-PORT").trim()
+        db_port = keyVault.find("${dataBasetName}-POSTGRES-PORT").trim()
         echo "retrieved db port: ${db_port}"
-        db_pass = keyVault.find("ccd-data-store-api-POSTGRES-PASS").trim().replaceAll('&', '\\\\&')
-        db_user = keyVault.find("ccd-data-store-api-POSTGRES-USER").trim()
+        db_pass = keyVault.find("${dataBasetName}-POSTGRES-PASS").trim().replaceAll('&', '\\\\&')
+        db_user = keyVault.find("${dataBasetName}-POSTGRES-USER").trim()
         echo "retrieved db user: ${db_user}"
-        db_name = keyVault.find("ccd-data-store-api-POSTGRES-DATABASE").trim()
+        db_name = keyVault.find("${dataBasetName}-POSTGRES-DATABASE").trim()
         echo "retrieved db name: ${db_name}"
         db_url = "jdbc:postgresql://${db_host}:${db_port}/${db_name}?ssl=true"
 
-        es_url = keyVault.find("ccd-ELASTIC-SEARCH-DATA-NODES-URL").trim()
+        es_url = getESUrlFromCCDVault(env)
         echo "retrieved ES URL: ${es_url}"
 
         sh "cp packer_images/${productName}_logstash.conf.in packer_images/${productName}_logstash.conf"
-        sh "sed -i 's|\${DB_URL}|${db_url}|' packer_images/${productName}_logstash.conf"
-        sh "sed -i 's|\${DB_USER}|${db_user}|' packer_images/${productName}_logstash.conf"
-        sh "sed -i 's|\${DB_PWD}|${db_pass}|' packer_images/${productName}_logstash.conf"
-        sh "sed -i 's|\"\${ES_DATA_NODES_URL}\"|${es_url}|' packer_images/${productName}_logstash.conf"
+        sh "sed -i 's|DB_URL|${db_url}|' packer_images/${productName}_logstash.conf"
+        sh "sed -i 's|DB_USER|${db_user}|' packer_images/${productName}_logstash.conf"
+        sh "sed -i 's|DB_PWD|${db_pass}|' packer_images/${productName}_logstash.conf"  // need to fix & isssue in password
+        sh "sed -i 's|ES_DATA_NODES_URL|${es_url}|' packer_images/${productName}_logstash.conf"
+         //Not working in sprod
+        // sh "sed -i 's|\${DB_URL}|${db_url}|' packer_images/${productName}_logstash.conf"   
+        // sh "sed -i 's|\${DB_USER}|${db_user}|' packer_images/${productName}_logstash.conf"
+        // sh "sed -i 's|\${DB_PWD}|${db_pass}|' packer_images/${productName}_logstash.conf"
+        // sh "sed -i 's|\"\${ES_DATA_NODES_URL}\"|${es_url}|' packer_images/${productName}_logstash.conf"
```

Comment:
CCD and Probate need to be aligned here, I suspect the var naming has moved on in CCD and Probate simply needs to be aligned with that.  @mario-paniccia probably has clearer idea on that.  Same goes for the other two instances below.

Reply:
The Logic was not working in my saat test. It did not replace values.